pages/contact.tsx

```diff
@@ -0,0 +1,328 @@
+/* eslint-disable react/no-children-prop */
+import React, { useState } from 'react'
+
+import {
+  Box,
+  Button,
+  Flex,
+  FormControl,
+  FormHelperText,
+  FormLabel,
+  Heading,
+  HStack,
+  Input,
+  InputGroup,
+  InputLeftElement,
+  Text,
+  Textarea,
+  VStack,
+  Wrap,
+  WrapItem,
+} from '@chakra-ui/react'
+import { GetStaticProps } from 'next'
+import { serverSideTranslations } from 'next-i18next/serverSideTranslations'
+import { useTranslation } from 'react-i18next'
+import { BsPerson } from 'react-icons/bs'
+import { MdEmail, MdLocationOn, MdPhone } from 'react-icons/md' //MdOutlineEmail
+
+import { Container, Layout, SocialButtons } from '@components'
+
+const Contact = (): JSX.Element => {
+  const { t } = useTranslation()
+  const [fullname, setFullname] = useState('')
+  const [email, setEmail] = useState('')
+  const [message, setMessage] = useState('')
+  /*
+const res = await fetch(` https://samenvvvv.com/email`, {
+            method: 'POST',
+            body: JSON.stringify({
+              to: 'info@samenvvv.nl',
+              from: "no-reply@samenvvvv.com",
+              replyTo: email,
+              subject: 'CalculatAR Report',
+              html: "<p>Here is example html</p>",
+            text:message,
+            headers: {
+              'Content-Type': 'application/json',
+            },
+          }),
+        })
+*/
+  //   Form validation
+  const [errors, setErrors] = useState<any>({})
+
+  //   Setting button text
+  const button_text = t('contact_message_send')
+  const [buttonText, setButtonText] = useState(button_text)
+
+  const [showSuccessMessage, setShowSuccessMessage] = useState(false)
+  const [showFailureMessage, setShowFailureMessage] = useState(false)
+
+  const handleValidation = () => {
```

Comment:
We will probably use forms in the future, for example competition application form. 

1. We can create reusable form elements (Input, Textarea) with validation states.
2. There are some popular form libraries which allow to make better forms in React. I think we will need one for more complex forms in the future.
  - [react hook form](https://github.com/react-hook-form/react-hook-form)
  - [formik](https://github.com/formium/formik)
  - [react form](https://github.com/tannerlinsley/react-form)